event_registration_cancel_reason/readme/CREDITS.rst

```diff
@@ -0,0 +1,5 @@
+Authors
+~~~~~~~
+
+* Tecnativa
+* Antiun Ingeniería S.L.
```

Comment:
No need to put this file. Authors are put automatically by the script. This file is intended for extra credits, like funders or so.